iati/resources.py

```diff
@@ -184,12 +189,48 @@ def get_codelist_path(codelist_name, version=None):
     return get_path_for_version(os.path.join(PATH_CODELISTS, '{0}'.format(codelist_name) + FILE_CODELIST_EXTENSION), version)
 
 
+def get_codelist_mapping_path(version=None):
+    """Determine the path of the Codelist mapping file.
+
+    version (str): The version of the Standard to return the data files for. Defaults to None. This means that the path is returned for a filename at the latest version of the Standard.
+
+    Returns:
+        str: The path to a file containing the mapping file.
+
+    Todo:
+        Test this.
```

Comment:
Why isn't this tested?

Reply:
https://github.com/IATI/pyIATI/commit/fca1fb0864057049606b6798ab351177e57e038f